### PR TITLE
.Q.trp uses 'trap at' rather than 'trap'

### PR DESCRIPTION
**Pull request by sshanks-kx**

### CODE REVIEW

**Comment by sshanks-kx** [comment]
addition of '4.1 addition of .Q.trpd' change to prevent conflicts

File: docs/ref/dotq.md [hunk 1]
```diff
@@ -1803,6 +1802,47 @@ Since V3.5 2017.03.15.
...
 
+## `trpd` (extend trap)
+
+```syntax
+.Q.trpd[f;x;g]
+```
+
+Where
+
+-   `f` is a function of rank and `x` an atom or list of count with items in the domains of f
```
Comment (JosieMD) [inline docs/ref/dotq.md]: @sshanks-kx can you clarify this line? Should it be two lines? Like this:
```suggestion
-   `f` is a function of rank
-   `x` is an atom or list of count with items in the domains of f

```

File: docs/ref/dotq.md [hunk 2]
```diff
@@ -1803,6 +1802,47 @@ Since V3.5 2017.03.15.
...
+-   `g` is a binary function
+
+extends [Trap](apply.md#trap) (`.[f;x;g]`) to collect backtrace: `g` gets called with arguments:
```
Comment (JosieMD) [inline docs/ref/dotq.md]:
```suggestion
extends [Trap](apply.md#trap) (`.[f;x;g]`) to collect backtrace: `g` is called with arguments:
```

File: docs/ref/signal.md [hunk 3]
```diff
@@ -69,10 +69,10 @@ At any point during execution, the behavior of _signal_ (`'`) is determined by t
 <div markdown="1" class="typewriter">
 0   abort execution (set by [Trap or Trap At](apply.md#trap)) 
 1   suspend execution and run the debugger
-2   collect stack trace and abort (set by [.Q.trp](dotq.md#trp-extend-trap))
+2   collect stack trace and abort (set by [.Q.trp](dotq.md#trp-extend-trap-at))
 </div>
 
-During abort, the stack is unwound up to the nearest [trap](apply.md#trap) (`@` or `.` or [`.Q.trp`](dotq.md#trp-extend-trap)). The error-trap mode is always initially set to 
+During abort, the stack is unwound up to the nearest [trap at](apply.md#trap-at) (`@` or `.` or [`.Q.trp`](dotq.md#trp-extend-trap-at)). The error-trap mode is always initially set to
```
Comment (JosieMD) [inline docs/ref/signal.md]:
```suggestion
During abort, the stack is unwound as far as the nearest [trap at](apply.md#trap-at) (`@` or `.` or [`.Q.trp`](dotq.md#trp-extend-trap-at)). The error-trap mode is always initially set to 
```

**Review by JosieMD — COMMENTED** [review]
@sshanks-kx a few comments/questions for you.

**Comment by sshanks-kx** [comment]
changes made, thanks

**Pull request merged**